modules/govuk_jenkins/templates/config/config.xml.erb

```diff
@@ -167,15 +164,21 @@
           <default>
             <comparator class="hudson.util.CaseInsensitiveComparator"/>
           </default>
+          <%- if @aws_migration -%>
```

Comment:
I'm not sure if we need this and the lines below it

Reply:
No idea where these came from... 😕